ecosystem.config.json

```diff
@@ -0,0 +1,13 @@
+{
+  "apps":
+    [
+      {
+        "script": "./src/index.js",
+        "name": "memebot-api",
+        "env": { "NODE_ENV": "production", "MEME_DIR": "./.memes" },
+        "max_memory_restart": "900M",
+        "exp_backoff_restart_delay": 100,
+        "max_restarts": 10
```

Comment:
Same general question as the memebot pull request on why the max restarts.

Reply:
Also this is just if it crashes almost immediately after startup 10 times in a row.